srcpkgs/upower/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'upower'
 pkgname=upower
-version=0.99.7
-revision=4
+version=0.99.9
+revision=1
+_release_sha=2282c7c0e53fb31816b824c9d1f547e8
 build_style=gnu-configure
 configure_args="--disable-static --with-backend=linux"
 hostmakedepends="pkg-config intltool libxslt docbook-xsl $(vopt_if gir gobject-introspection)"
 makedepends="libusb-devel libgudev-devel dbus-glib-devel libimobiledevice-devel polkit-devel"
 conf_files="/etc/UPower/UPower.conf"
 short_desc="Abstraction for enumerating power devices"
-maintainer="Juan RP <xtraeme@voidlinux.eu>"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
 license="GPL-2.0-or-later"
-homepage="http://upower.freedesktop.org"
-distfiles="${homepage}/releases/${pkgname}-${version}.tar.xz"
-checksum=24bcc2f6ab25a2533bac70b587bcb019e591293076920f5b5e04bdedc140a401
+homepage="https://upower.freedesktop.org"
+# upstream doesn't update the relase tarballs :c
```

Comment:
Freedesktop is moving to gitlab. Please update `update` too if this will be merged and not #5695.

Reply:
Oh, right. Didn't see your PR. I guess we can go with your PR, although I'd recommend that you use the release and not the tag for distfiles